test/guest/debian-8.setup

```diff
@@ -0,0 +1,102 @@
+#! /bin/bash
+
+set -ex
+
+# TODO: Hack "console=ttyS0,115200" into kernel command line
+
+# We install all dependencies of the cockpit packages since we want
+# them to not spontaneously change from one test run to the next when
+# the distribution repository is updated.
+#
+COCKPIT_DEPS="\
+docker.io \
+libjson-glib-1.0-0 \
+libpcp3 \
+libpolkit-agent-1-0 \
+libpolkit-gobject-1-0 \
+libpwquality-tools \
+libssh-4 \
+network-manager \
+pcp \
+policykit-1 \
+realmd \
+selinux-basics \
+xdg-utils \
+"
+
+# XXX - We have a explicit list of build dependencies here since that
+# there is no straighforward way to have pbuilder install them.
+#
```

Comment:
We want to pull these in advance, and not rely in build time network access. These builds happen hundreds of times a day ... so downloading packages every time makes them slow and flakey.